ocaml/xapi/sm_exec.ml

```diff
@@ -341,7 +341,6 @@ let with_session sr f =
 let exec_xmlrpc ~dbg ?context:_ ?(needs_session = true) (driver : string)
     (call : call) =
   with_dbg ~name:call.cmd ~dbg @@ fun di ->
```

Comment:
IIUC, `with_dbg` will create a `Debuginfo.t` from the string `call.cmd`, only if it can be separated into two lines with the second line equal to the traceparent. This means that the `call.cmd` should have two lines in it and it is the caller's responsibility to set things up so that a tracing is created?

Reply:
'with_dbg' will create a 'Debuginfo.t' from 'dbg'. And 'dbg' is made of 2 lines: logs and tracing. `name` only gets appended to the span name.